kustomize/commands/openapi/fetch/fetch.go

```diff
@@ -36,15 +48,25 @@ Installation and setup instructions: https://kubernetes.io/docs/tasks/tools/inst
 	command.Stdout = &stdout
 	command.Stderr = &stderr
 	err := command.Run()
-	if err != nil || stdout.String() == "" {
-		fmt.Fprintln(w, err, stderr.String()+errMsg)
-		return
+	if err != nil {
+		return fmt.Errorf(err.Error(), stderr.String()+errMsg)
```

Comment:
When I tried this out locally, I got a bit of a weird error output:

```
Error: exit status 1%!(EXTRA string=The connection to the server localhost:8080 was refused - did you specify the right host or port?

Error fetching schema from cluster.
Please make sure kubectl is installed, its context is set correctly, and your cluster is up.
Installation and setup instructions: https://kubernetes.io/docs/tasks/tools/install-kubectl/)
```

I think what we want here is to use the `%w` verb to identify the error, something like:

```go
		return fmt.Errorf("%w\n%s", err, stderr.String()+errMsg)
```

Will give:

```
Error: exit status 1
The connection to the server localhost:8080 was refused - did you specify the right host or port?

Error fetching schema from cluster.
Please make sure kubectl is installed, its context is set correctly, and your cluster is up.
Installation and setup instructions: https://kubernetes.io/docs/tasks/tools/install-kubectl/
```

Reply:
done! thank you